script.js

```diff
@@ -1,12 +1,236 @@
+
+    let runtime = document.createElement('div')
+    runtime.classList.add('runtime')
+    runtime.innerText = "runtime: " + episode.runtime + " min"
+        
+    divCard.appendChild(div)
+    divCard.appendChild(imgCard)
+    divCard.appendChild(divCardSummery)
+    divCard.appendChild(runtime)
+
+    rootElem.appendChild(divCard)
+
+    pCount.innerText =`${arr.length}/${arr.length} episodes`
+  }
+
+  async function loadEpisodes(id) {
+    let url = `https://api.tvmaze.com/shows/${id}/episodes`;
+    let obj = await (await fetch(url)).json();
+    episodes = obj
+    
+    episodes.forEach(addEpisodes)
+    
+    pCount.innerText =`${episodes.length}/${episodes.length} episodes`
+  }
+
+
+
+
+  
+  //Select field changing options
+    
+  select.addEventListener('change', () => {
+
+    if(select.value == "allepisodes"){
+      rootElem.innerHTML =""
+      arr.forEach(addShows)
+      pCount.innerText =`${arr.length}/${arr.length} episodes`
+    } else {
+
+    let id = select.value
+    rootElem.innerHTML =""
+
+    loadEpisodes(id);
+    }
+
+  })
+
+
+function addShows(episode){
+
+  let divCard = document.createElement('div')
+  divCard.classList.add('card')
+  let div = document.createElement('div')
+  div.classList.add('cardTitle')
+  if(episode.season){
+  div.innerText = `${episode.name} - ${titleCodeGenerator(episode)}`
+  } else {div.innerText = `${episode.name}`}
+  let imgCard = document.createElement('img')
+  imgCard.classList.add("cardImg")
+  imgCard.src=`${episode.image !== null ? episode.image.medium : "https://www.facultatieve-technologies.com/wp-content/uploads/No-image-200px.png"}`
+  let divCardSummery = document.createElement('div')
+  divCardSummery.classList.add('cardSummary')
+  divCardSummery.innerText=`${episode.summary.replace(/<\/?[^>]+(>|$)/g, "")}`
+
+  let mainDiv = document.createElement('div')
+  mainDiv.classList.add('mainDiv')
+
+  let runtime = document.createElement('div')
+  runtime.classList.add('runtime')
+  runtime.innerText = "runtime: " + episode.runtime
+
+  let genres = document.createElement('div')
+  genres.classList.add('genres')
+  genres.innerText = "Genres: " + episode.genres.join('; ')
+
+  let status = document.createElement('div')
+  status.classList.add('status')
+  status.innerText = "status: " + episode.status
+
+  let rating = document.createElement('div')
+  rating.classList.add('rating')
+  rating.innerText = "rating: " + episode.rating.average
+
+  div.addEventListener('click', ()=> {
+
+    let id = episode.id
+    rootElem.innerHTML =""
+
+    loadEpisodes(id);
+    select.value = id
+
+  })
+
+
+    
+    
+    divCard.appendChild(div)
+    mainDiv.appendChild(imgCard)
+    mainDiv.appendChild(divCardSummery)
+    divCard.appendChild(mainDiv)
+    divCard.appendChild(genres)
+    divCard.appendChild(status)
+    divCard.appendChild(rating)
+    divCard.appendChild(runtime)
+
+
+
+
+    rootElem.appendChild(divCard)
 }
```

Comment:
it's good to use of functionality coding but it's better to use of short functions, it should be more readable and if you add some comment in your code it should help someone who wants to read your code.